development/docker-compose.base.yml

```diff
@@ -38,3 +38,13 @@ services:
       retries: 3
       test: ["CMD", "bash", "-c", "nautobot-server celery inspect ping --destination celery@$$HOSTNAME"]  ## $$ because of docker-compose
     <<: *nautobot-base
+  beat:
```

Comment:
Should this be "scheduler" or something more generic? Not important here, but for dev standards type question.

Reply:
This naming follows celery commands specified in `entrypoint`.